Makefile

```diff
@@ -145,7 +145,7 @@ fmt:
 	@gofmt -l -w $(SOURCE_DIRS)
 
 $(GOPATH)/bin/golangci-lint:
-	go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.27.0
+	pushd /tmp && GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.27.0 && popd
```

Comment:
quite ugly :(( Is GO111MODULE required?

Reply:
yes without it, if you have $GOPATH set as default, it will complain about it and exit

```
go: cannot use path@version syntax in GOPATH mode
make: *** [Makefile:148: /home/prkumar/work/go/bin/golangci-lint] Error 1
```